linc_convert/modalities/df/multi_slice.py

```diff
@@ -247,17 +251,17 @@ def convert(
                 "scale": [1.0] * has_channel
                 + [
                     1.0,
-                    (shape0[0] / shape[0]) * vxh,
-                    (shape0[1] / shape[1]) * vxw,
+                    float(2**n) * vxh,
+                    float(2**n) * vxw,
                 ],
             },
             {
                 "type": "translation",
                 "translation": [0.0] * has_channel
                 + [
                     0.0,
-                    (shape0[0] / shape[0] - 1) * vxh * 0.5,
-                    (shape0[1] / shape[1] - 1) * vxw * 0.5,
+                    float(2**n - 1) * vxh * 0.5,
+                    float(2**n - 1) * vxw * 0.5,
```

Comment:
Can you explain what the issue with webknossos is?

The thing it that you may be fixing things for webknossos, but it actually mis-registers the layer (if we're following the spec) and will break things down in our own code in the future. So I am a bit against this change.